lib/rspec-puppet.rb

```diff
@@ -48,6 +48,7 @@ def self.rspec_puppet_example?
   c.add_setting :derive_node_facts_from_nodename, :default => true
   c.add_setting :adapter
   c.add_setting :platform, :default => Puppet::Util::Platform.actual_platform
+  c.add_setting :trusted_server_facts, :default => false
```

Comment:
One thing I've spotted is that this config item has been deprecated in Puppet 4.10, so not sure of the best way to handle it... 

Thoughts @rodjek ?